recipes/fltk/all/conanfile.py

```diff
@@ -11,7 +11,7 @@
 class FltkConan(ConanFile):
     name = "fltk"
     description = "Fast Light Toolkit is a cross-platform C++ GUI toolkit"
-    license = "LGPL-2.0-custom"
+    license = "LGPL-2.1+ WITH FLTK-exception"
```

Comment:
```suggestion
    license = "LGPL-2.1-or-later WITH FLTK-exception"
```

Reply:
The plus notation is supported by the SPDX expression syntax: https://spdx.github.io/spdx-spec/v2-draft/SPDX-license-expressions/#d3-simple-license-expressions
It's what their online tool spat out for me and I think it's the preferred format thanks to being more generally applicable.
As a data point, it's the format used in the Linux kernel: https://docs.kernel.org/process/license-rules.html

Reply:
It's deprecated. You have to scroll to the bottom of https://spdx.org/licenses/ to see the deprecated licenses.

Reply:
Oh, thanks!